src/components/NavNebari.vue

```diff
@@ -0,0 +1,372 @@
+<template>
+  <div>
+    <div class="dropdown">
+      <button
+        @click="downloadProject"
+        class="nebari-button external-links"
+        title="Open in Nebari server instance"
+      >
+        <div>
+          <!-- Icon like in GH -->
+          <span class="topline">
+            <img
+              src="../assets/Nebari-logo-square.svg"
+              height="18"
+              viewBox="0 0 18 18"
+              version="1.1"
+              width="18"
+            />
+          </span>
+          <span id="code">Open in Nebari</span>
+          <span class="topline">
+            <svg
+              aria-hidden="true"
+              height="16"
+              viewBox="0 0 16 16"
+              version="1.1"
+              width="16"
+              data-view-component="true"
+            >
+              <path
+                d="m4.427 7.427 3.396 3.396a.25.25 0 0 0 .354 0l3.396-3.396A.25.25 0 0 0 11.396 7H4.604a.25.25 0 0 0-.177.427Z"
+              ></path>
+            </svg>
+          </span>
+        </div>
+      </button>
+      <div class="dropdown-content">
+        <div class="text-box-nebari" id="nebari-text">
+          <span class="sub-text">
+            <a href="https://www.nebari.dev/"><strong>Nebari</strong></a
+            >: open source data science platform
```

Comment:
Let's make all the text as a link
```suggestion
            <a href="https://www.nebari.dev/">
              <strong>Nebari</strong>: open source data science platform
            </a>  
```